cmd/core-command/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/dumb-init /bin/sh
+#  ----------------------------------------------------------------------------------
+#  Copyright (c) 2021 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#  ----------------------------------------------------------------------------------
+
+set -e
+
+echo "CONSUL_HOST: ${CONSUL_HOST} CONSUL_PORT: $CONSUL_PORT"
+
+echo "$(date) Starting edgex-core-command..."
+exec /core-command -cp=consul.http://"${CONSUL_HOST}":"$CONSUL_PORT" --registry --confdir=/res
```

Comment:
Change to `exec /core-command $@`. Same for all other entry point scripts.

Reply:
Will the entry point script be essentially the same for each service? If so, we should inject a common script like with the non-edgex services. Use '$@` to know what to execute. So `ENTRYPOINT` and `CMD` would remain as they are today and be overridden in the secure compose files.

Reply:
yes, it is the same, all other core-services are injected by `security-bootstrapper` via `others_wait_install.sh` (https://github.com/jim-wang-intel/edgex-go/blob/security-bootstrapper/cmd/security-bootstrapper/entrypoint-scripts/others_wait_install.sh).

Reply:
This entrypoint.sh was placed here for non-secty docker-compose ; so there is a chance to move this logic into docker-compose builder.

Reply:
As we discussed earlier these are not needed for non-secty. Please remove.,

Reply:
removed.